2018/05.md

```diff
@@ -85,81 +86,85 @@ Supporting materials includes slides, a link to the proposal repository, a link
-        1. Binary AST update (Shu-yu Guo, Eric Faust, Vladan Djeric) ([slides](https://docs.google.com/presentation/d/12A0w7XuazDyhvSxqPcfXZCqV17iFJF6LOaNP-gtzifM/edit?usp=sharing), [spec rough draft](https://binast.github.io/ecmascript-binary-ast/))
-        1. Update to the How We Work Documentation project and [Website](https://github.com/tc39/tc39-web-draft)
-1. Non-timeboxed overflow from previous meeting
-1. Non-timeboxed agenda items
-    1. Stage 0+ proposals looking to advance
-    1. New proposals
-    1. Discussion and updates for Stage 0+ Proposals
-        1. Open-ended discussion: How should we analyze complexity and cross-cutting concerns on syntax and library proposals? (Daniel Ehrenberg) (timebox: 30m)
-        1. Open-ended discussion: How should we collaborate within a large committee? (Daniel Ehrenberg) (timebox: 30m)
-1. Overflow from timeboxed discussion items (in insertion order)
+
+    | timebox | topic | presenter |
+    |:-------:|-------|-----------|
+    | 15m     | [Normative: Cleanup Time Values and Time Range](https://github.com/tc39/ecma262/pull/1144) Needs Consensus PR | Andrew Paprocki |
+    | 15m     | [Normative: Add `export * as ns from "mod”` to Export production and Module Semantics](https://github.com/tc39/ecma262/pull/1174) | Valerie Young, John-David Dalton |
+    | 15m     | Array.prototype.values web compat update | Sathya Gunasekaran |
+
+1. Overflow from previous meeting
+
+    | timebox | topic | presenter |
+    |:-------:|-------|-----------|
+
+1. Short (&le;30m) Timeboxed Discussions
+
+    | timebox | topic | presenter |
+    |:-------:|-------|-----------|
+    | 15m     | [2019](https://github.com/tc39/Reflector/issues/130)/2020 meeting scheduling update | Daniel Ehrenberg |
+    | 15m     | Update to the How We Work Documentation project and [Website](https://github.com/tc39/tc39-web-draft) | yulia (@codehag) |
+    | 30m     | Intl update ([slides](https://docs.google.com/presentation/d/1wEkpdxC37t4sk64QThcna8c4753-9Ak1I23LNDmZ9KE/edit?usp=sharing)) | Daniel Ehrenberg |
+    | 30m     | [Rename Atomics.wake](https://docs.google.com/presentation/d/1awJqO2WcBs4UlBaLZn0mfwV3xOYOlo0hHNbB6mtctnQ/) | Rick Waldron, Filip Pizlo, Leo Balter |
+    | 30m     | [Supporting other languages in ES module graphs](https://linclark.github.io/wasm-es-modules/slides/2018-05-23/#/0) updates | Lin Clark |
+
+1. Proposals
+
+    | stage | timebox | topic | presenter |
+    |:-----:|:-------:|-------|-----------|
+    | 3     | 15m     | [Optional catch binding](https://tc39.github.io/proposal-optional-catch-binding/) for stage 4 ([#1185](https://github.com/tc39/ecma262/pull/1185)) | Michael Ficarra |
+    | 3     | 15m     | [ECMAScript ⊃ JSON](https://github.com/tc39/proposal-json-superset) for Stage 4 ([#1188](https://github.com/tc39/ecma262/pull/1188)) | Mathias Bynens |
```

Comment:
also this pr link

Reply:
First I hear of this requirement. Thanks, @michaelficarra!

Reply:
see #311

Reply:
@mathiasbynens Having the PR open is the stage 4 requirement. Linking to it from the agenda is just proposed in #311 for now, as @ljharb points out. Sorry, my wording was ambiguous.

Reply:
suffice to say it's not a requirement yet, but it's still useful to do :-) your suggestion is one way to resolve #311 indeed.